client/src/components/NavBar/index.tsx

```diff
@@ -6,7 +6,7 @@ import { EntityContext } from '../../context/CommitteeContext'
 import { getEmptyEntity } from '../../utils/EmptyEntity'
 
 const NavBar = ({ data }: INavBar) => {
-  const [tabSelected, setTabSelected] = useState('')
+  const [tabSelected, setTabSelected] = useState('comittee')
```

Comment:
Here it should probably be useState('committee') instead of 'comittee' with only one 'm'